### DISCUSSION TITLE
Able to edit inline format to account for space before 2 colons?

### DISCUSSION THREAD

**Discussion by miscelly** [RafaelGB/obsidian-db-folder#503] [Q&A] [answered]
I can't find this anywhere but it doesn't mean there's no way to do it, so I thought I'd ask here. I have a few hundred files that I want to use this plugin with.  My inline values all have this format:

> Field :: Value

I always add a space between the colons just for better readability. I can see the data just fine in the database table, but if I edit a value using the plugin, the original doesn't change and I end up with this added to the file:

> Field:: Value

In the database table I'll see both values in the format of "firstvalue, secondvalue" within the one cell.

Is there any way I can prevent that from happening other than modifying a few hundred files? Thanks!

**Comment by RafaelGB** [comment, marked as answer]
hi @miscelly , I am afraid the plugin just allow the format of `key:: value` for editing. could accept the wrap of () and []

We can work on the regex expression to accept space (just 1 ) between key and dots

**Reply by DryIce1** [reply]
edit: moved my comment to https://github.com/RafaelGB/obsidian-db-folder/issues/556#issuecomment-1347075407

**Reply by RafaelGB** [reply]
I am already trying to support the edition of those cases, but this regex does not work.
On regex101 looks fine and I can not see the problem by now (still working on it)

<img width="1706" alt="Screenshot 2022-12-12 at 22 05 37" src="https://user-images.githubusercontent.com/11924043/207154438-d5747d6c-7e51-44d2-8b21-3c8209d768f1.png">

**Reply by RafaelGB** [reply]
https://regex101.com/r/5r3HOD/1

**Comment by miscelly** [comment]
No worries, I can change the format in the files. It's probably easier for me to change the format than it is for a code change to be made. Less risky anyway. Thanks for the quick response.

**Reply by RafaelGB** [reply]
anyway, to expand the possibilities I will add this request to the roadmap 😊
